pkg/plugins/trivy/jobspec.go

```diff
@@ -51,22 +51,6 @@ type PodSpecMgr interface {
 func NewPodSpecMgr(config Config) PodSpecMgr {
```

Comment:
Can we add a TODO to the callers of `NewPodSpecMgr()` to check if `err != nil`? Currently we do not and this will result in a panic here: https://github.com/aquasecurity/trivy-operator/blob/1c0f3d938fb16b3ab67c1599b2dccebde46a51bb/pkg/plugins/trivy/plugin.go#L110

Reply:
If there's an unsupported workload, this would return an empty structure https://github.com/aquasecurity/trivy-operator/blob/main/pkg/kube/object.go#L264

In such a case, should we check the error first prior to using podSpec?